network.go

```diff
@@ -62,6 +62,7 @@ type NetworkInfo interface {
 	IpamInfo() ([]*IpamInfo, []*IpamInfo)
 	DriverOptions() map[string]string
 	Scope() string
+	EnableIPv6() bool
```

Comment:
Thanks for the change.
Can we rename this to `IPv6Enabled()`. 
`EnableIPv6()` sounds more like an command, rather than a query.